### ISSUE TITLE
The DEnsemble model doesn't use the dvalid data.

### ISSUE THREAD

**Issue by wuzhe1234** [microsoft/qlib#1208]
There is no `early_stopping_round` parameter the config yaml file of double ensemble model. So according to the source code, each sub-model will only use dtrian data to train the sub-model for a specific number of epochs.  The data between "2015-01-01" and "2016-12-31" does not contribute to the model at all. Is it designed this way?

**Comment by you-n-g** [comment]
Here is a reference from the paper.

Here is the setting for LightGBM for baseline models, which DoubleEnsemble shares. The epoch is fixed.  
![image](https://user-images.githubusercontent.com/465606/179531934-e1969792-4eeb-4b5a-8ab3-ec558502c321.png)

Besides, we happened to discuss with the author and were told that the epoch is fixed.

**Comment by wuzhe1234** [comment]
Thank you so much for your response, I now understand that the epochs is fixed. And I noticed that only train dataset is uesed for retriving the loss curve and determining the weights. In order to prove it,  I randomly drop 50% of valid dataset. And the pred scores remain the same. Thanks again for your detailed answer！
